maintainers.md

```diff
@@ -6,8 +7,31 @@ To update this repo, run the following commands:
 pip install cookiecutter
 # Optionally replace `branch = None` with a particular branch name
 echo 'branch = None; from cookiecutter.main import cookiecutter; cookiecutter("https://github.com/zagaran/django-template", checkout=branch, extra_context={"project_slug": "sample_django_app", "feature_annotations": "on"}, overwrite_if_exists=True, no_input=True, output_dir="..")' | python3
-pip-compile --upgrade
+pip-compile
```

Comment:
This is an annoying change, but necessary; if you have both `django-ses` installed and any library that requires `boto3`, there is a high likelihood that pip-compile will fail, as `boto3` has insanely specific `botocore` requirements.

Reply:
most of the dependency clash seems to come from the aws CLI and EB CLI, which we've taken out of requirements